internal/core/conanfile.txt

```diff
@@ -0,0 +1,31 @@
+[requires]
+rocksdb/6.29.5
+boost/1.80.0
+onetbb/2021.3.0
+zstd/1.5.2
+arrow/8.0.1
+openssl/1.1.1q
+aws-sdk-cpp/1.9.234
+benchmark/1.7.0
+gtest/1.8.1
+protobuf/3.9.1
+rapidxml/1.13
+yaml-cpp/0.7.0
+marisa/0.2.6
+zlib/1.2.13
+
+[generators]
+cmake
+
+[options]
+rocksdb:shared=True
+arrow:parquet=True
+arrow:compute=True
+arrow:with_zstd=True
+aws-sdk-cpp:text-to-speech=False
+aws-sdk-cpp:transfer=False
```

Comment:
My laptop can not build with gmock, enable the following option works for me: 
gtest:build_gmock=False
Seems we don't use gmock anyway,

Reply:
👌🏻